test/test_genbank_sbol3_direct.py

```diff
@@ -1,21 +1,39 @@
 import unittest
 import sbol3
+from pathlib import Path
 from sbol_utilities.sbol_diff import doc_diff
-from sbol_utilities.sbol3_genbank_conversion import convert_genbank_to_sbol3, \
-    GENBANK_FILE_1, SBOL3_FILE_1
+from sbol_utilities.sbol3_genbank_conversion import GenBank_SBOL3_Converter, TEST_NAMESPACE
 
 class TestGenBank2SBOL3(unittest.TestCase):
+    # Create converter instance
+    converter = GenBank_SBOL3_Converter()
     def test_simple_file_1(self):
         """Test conversion of a simple genbank file with a single sequence"""
-        sbol3.set_namespace("https://testing.sbol3.genbank/")
-        TEST_OUTPUT_SBOL3 = SBOL3_FILE_1 + ".test" 
+        SAMPLE_GENBANK_FILE_1 = str(Path(__file__).parent) + "/test_files/" + "BBa_J23101.gb"
```

Comment:
This style of path appending won't work on all platforms. Please use the file append operations. If you are using `pathlib.Path` you can do it like this:
```python
Path(__file__).parent / 'test_files' / 'BBa_J23101.gb'
```
`pathlib.Path` defines the `/` operator to be the path append operation in a cross-platform way.